cookbooks/bcpc/templates/default/cobbler/dhcp.template.erb

```diff
@@ -86,7 +86,7 @@ group {
 	  if option pxe-system-type = 00:00 {
 	    filename "/pxelinux.0";
 	  } else {
-   	    filename "http://$next_server/cblr/svc/op/gpxe/system/$iface.hostname";
+   	    filename "http://$next_server/cblr/svc/op/gpxe/system/$iface.owner";
```

Comment:
Cobbler 2.6 changed the behavior of default host attributes.

I have no idea what `owner` means in context but it contains the value we were expecting.